main.go

```diff
@@ -1240,6 +1248,10 @@ func main() {
 			Name:  "rr-dns-mode",
 			Usage: "enable round-robin DNS mode",
 		},
+		cli.BoolFlag{
+			Name:  "optimistic",
```

Comment:
can we rename this flag as 

```suggestion
			Name:  "health-optimistic",
```